sql/mysql_db_structure.sql

```diff
@@ -138,11 +142,11 @@ CREATE TABLE `contestproblem` (
 --
 
 CREATE TABLE `contestteam` (
-  `cid` int(4) unsigned NOT NULL COMMENT 'Contest ID',
-  `teamid` int(4) unsigned NOT NULL COMMENT 'Team ID',
+  `cid` int(4) unsigned NOT NULL COMMENT 'Unique contest ID',
+  `teamid` int(4) unsigned NOT NULL COMMENT 'Unique team ID',
   PRIMARY KEY (`cid`,`teamid`),
-  KEY `cid` (`cid`),
-  KEY `teamid` (`teamid`),
+  KEY `IDX_8328F8554B30D9C4` (`cid`),
```

Comment:
For ManyToMany join tables we can not really influence the name of keys. Doctrine uses a hash of the fields for this.